spec/jobs/scrapping_job_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe ScrappingJob, :type => :job do
+  describe "perform_later" do
+    it "having job enqueued" do
+      ActiveJob::Base.queue_adapter = :test
```

Comment:
Kindly move it to one file which we use to set configurations of test environment.

Reply:
in 'specs/rails_helper' , i have set the enviornment variable as ActiveJob::Base.queue_adapter = :test .
issue resolved .